task/python-boto3/0.1/python-boto3.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
since `tekton.dev/v1` is now available, can we use that instead of `v1beta1` ?

Reply:
done